src/parthenon_manager.cpp

```diff
@@ -131,6 +131,10 @@ ParthenonStatus ParthenonManager::ParthenonInitEnv(int argc, char *argv[]) {
   Globals::receive_boundary_buffer_timeout =
       pinput->GetOrAddReal("parthenon/time", "recv_bdry_buf_timeout_sec", -1.0);
 
+  // set boundary comms buffer switch trigger
+  Globals::cell_centered_refinement::min_num_bufs =
+      pinput->GetOrAddReal("parthenon/mesh", "refinement_in_one_min_nbufs", 6);
+
```

Comment:
This number should probably be higher. An A100 has up to 128 SMs but I don't have a feeling for where the tradeoff between launching kernels and runtime is for small block sizes.
For small blocks this number is easily in the thousands on a device and for large blocks the latency will be less important.
My gut tells me that sth like 64 might be a good start but as this is a runtime parameter we can leave as is and remind people that this may impact performance.
Having said that I'm wondering it'd make sense to start a doc page to highlight the most important performance knobs for a given simulation, e.g., pack_size, number of ranks per GPU, this parameter, etc.